.github/workflows/CI.yml

```diff
@@ -21,7 +21,6 @@ jobs:
           - zulu
         java-version:
           - 8
-          - 11
```

Comment:
Do the tests still break on Java 11 with the dependency changes?
If not would it make sense to re-add it to the test matrix?

Reply:
I'm re-adding Java 11 back to this file. I found that there were 2 dependency upgrades that were causing the tests to fail, despite it working after performing a dataInit, portalInit, and tomcatDeploy.